File: PRAK110-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
#include <math.h>
int main () {

    int alas = 5;
    int tinggi = 12;
    int sisi_miring =sqrt(tinggi*tinggi + alas*alas);

    printf ("Diketahui : \n");
    printf ("Alas = %d cm\n", alas);
    printf ("Tinggi = %d cm\n", tinggi);
    printf ("\n");
    printf ("Jawab : \n");
    printf ("Sisi A = %d cm\n", tinggi);
    printf ("Sisi B = %d cm\n", sisi_miring);
    printf ("Sisi C = %d cm\n", alas);
    printf ("Keliling = %d cm\n", alas + tinggi + sisi_miring);
    printf ("Luas = %d cm\n", alas * tinggi / 2);
}

File: PRAK110-2210817120007-NursyahnaPuteri.py
import math 

alas = 5
tinggi = 12
sisi_miring = (math.sqrt(tinggi*tinggi + alas*alas))

print ("Diketahui :")
print ("Alas = {} cm".format (alas))
print ("Tinggi = {} cm".format (tinggi))
print ("Jawab : ")
print ("\n")
print ("Sisi A = {} cm" .format(tinggi))
print ("Sisi B = {}cm". format (round(sisi_miring,)))
print ("Sisi C = {}cm". format (alas))
print ("Keliling = {}cm".format (round(alas + tinggi + sisi_miring,)))
print ("Luas = {}cm". format (round(alas * tinggi/2,)))